apps/condo/domains/organization/integrations/sbbol/oauth2.js

```diff
@@ -2,29 +2,30 @@
 
 const { Issuer, custom } = require('openid-client') // certified openid client will all checks
 const jwtDecode = require('jwt-decode') // decode jwt without validation
-const dayjs = require('dayjs')
-
+const pino = require('pino')
+const falsey = require('falsey')
+const util = require('util')
 const conf = require('@core/config')
 
-const SBBOL_CONFIG = conf.SBBOL_CONFIG ? JSON.parse(conf.SBBOL_CONFIG) : {}
+const SBBOL_AUTH_CONFIG = conf.SBBOL_AUTH_CONFIG ? JSON.parse(conf.SBBOL_AUTH_CONFIG) : {}
```

Comment:
I don't think that's a good idea to change the name of config. You need to copy it on dev and prod. Don't forget it!

Reply:
I know, we have discussed, that finally we will have two config variables.